src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,133 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class Book {
```

Comment:
JPA needs an empty constructor to instantiate the class

Reply:
I thnik that this is resolved